Modules/Quotas/src/Quotas.Application/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -13,9 +10,12 @@ public static class IServiceCollectionExtensions
  
     public static void AddApplication(this IServiceCollection services)
     {
-        // services.AddMediatR(typeof(...)); // needs to be filled as soon as there is the first command/query
-        services.TryAddTransient(typeof(IPipelineBehavior<,>), typeof(LoggingBehavior<,>));
-        services.TryAddTransient(typeof(IPipelineBehavior<,>), typeof(RequestValidationBehavior<,>));
+        /*services.AddMediatR(c => c
+            .RegisterServicesFromAssemblyContaining<...>()
+            .AddOpenBehavior(typeof(LoggingBehavior<,>))
+            .AddOpenBehavior(typeof(RequestValidationBehavior<,>))
+            .AddOpenBehavior(typeof(QuotaEnforcerBehavior<,>))
+        );*/
```

Comment:
Why is this commented out?

Reply:
We cannot call `AddBehavior` whithout calling `RegisterServicesFromAssemblyContaining` first, which we're not doing.